app/src/components/MinMaxSlider.css

```diff
@@ -0,0 +1,101 @@
+.container {
```

Comment:
#### Non-blocking
It's not necessary to fix this in this PR, I would prefer the styles to be written as vanilla extract modules for the sake of consistency.
https://vanilla-extract.style/